src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx

```diff
@@ -56,11 +56,15 @@ Those default settings would be under one "authentication domain." Another authe
 
 When you add users to New Relic, they’re added to a specific authentication domain. Typically organizations have either one or two authentication domains: one for the manual, default methods and one for the methods associated with an identity provider.
 
-## Find the authentication domain UI [#ui]
+## Create and configure an authentication domain [#ui]
 
-If you meet the [requirements](#requirements), you can add and manage authentication domains. To do this: from the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary#account-dropdown), click **Organization and access**, and then click **Authentication domains**.
+If you meet the [requirements](#requirements), you can add and manage authentication domains. T
 
-For more about the configuration options, keep reading.
+To view and configure authentication domains: from the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary#account-dropdown), click **Organization and access**, and then click **Authentication domains**.
+
+Your existing domains will be on the left. Note that most organizations have, at most, two domains: one with the manual, default settings and one for the identity provider-related settings. 
```

Comment:
"Your existing domains are on the left..." or are they not there yet? Or that's where they'll be once a domain has been added?